build/downloadTypos.js

```diff
@@ -0,0 +1,196 @@
+// @ts-check
+/**
+ * OSに合ったtyposのバイナリをダウンロードするスクリプト。
+ */
+const { exec } = require("child_process");
+const { promisify } = require("util");
+const { platform, arch } = require("os");
+const { join, resolve } = require("path");
+const {
+  mkdirSync,
+  existsSync,
+  unlinkSync,
+  createWriteStream,
+  rmSync,
+} = require("fs");
+const fetch = require("node-fetch");
+
+// OS名を定義するオブジェクト
+const OS = {
+  LINUX: "linux",
+  MACOS: "darwin",
+  WINDOWS: "win32",
+};
+// CPUアーキテクチャ名を定義するオブジェクト
+const CPU_ARCHITECTURE = {
+  X86_64: "x86_64",
+  ARM: "aarch64",
+};
+// 全バイナリのパス
```

Comment:
ここだけちょっと事実と違うかも？　こうかな～と

```suggestion
// ダウンロードしたバイナリを格納するディレクトリ
```